immuni_analytics/celery/authorization_android/__init__.py

```diff
@@ -0,0 +1,13 @@
+#   Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
```

Comment:
I noticed an issue in the copyright plugin. As you can see it's adding `three` spaces after the `#`. If you check the template you gave, it should have `two`. What I did was changing the template to only have `one` (which is the original intention, btw) and re-run it.

The same applies to all the other copyrights added in this PR.